sys/include/atomic_utils.h

```diff
@@ -753,9 +774,31 @@ static inline atomic_bit_u32_t atomic_bit_u32(volatile uint32_t *dest,
  * @brief   Create a reference to a bit in an `uint64_t`
  * @param[in]       dest        Memory containing the bit
  * @param[in]       bit         Bit number (`0` refers to the least significant)
+ *
+ * @retval Opaque reference to the bit.
  */
 static inline atomic_bit_u64_t atomic_bit_u64(volatile uint64_t *dest,
                                               uint8_t bit);
+
+/**
+ * @brief   Create a reference to a bit in an `unsigned int`
+ * @param[in]       dest        Memory containing the bit
+ * @param[in]       bit         Bit number (`0` refers to the least significant)
+ *
+ * @retval Opaque reference to the bit.
```

Comment:
```suggestion
 * @return Opaque reference to the bit.
```

`@retval` is used to generate a table listing special return values (such as error codes) - one table row per `@retval` statement -, while `@return` is the description of the return value.